src/views/Home.jsx

```diff
@@ -17,15 +20,52 @@ export function Home({ createToken, setListToken }) {
 		} else {
 			listId = null;
 			setListToken(listId);
-			setMessage('Your shopping list was not created. Please try again. ');
+			setCreateListMessage(
+				'Your shopping list was not created. Please try again. ',
+			);
 		}
 	}
 
+	async function handleJoinClick(e) {
+		e.preventDefault();
+
+		if (!tokenInput) {
+			setExistingListMessage('Please enter a token.');
+		}
```

Comment:
**No need to change** I assume that if `tokenInput` is empty, we don't want to execute the rest of the function. If that's the case, I suggest you adding a `return` statement to make sure it ends when that happens.

```js
if (!tokenInput) {
	setExistingListMessage('Please enter a token.');
    return
}
```

Reply:
Thanks for the suggestion, we added the return.